sdk/identity/identity/src/credentials/deviceCodeCredential.ts

```diff
@@ -8,7 +8,7 @@ import { AuthenticationError, AuthenticationErrorName } from "../client/errors";
 import { createSpan } from "../util/tracing";
 import { delay } from "../util/delay";
 import { CanonicalCode } from "@opentelemetry/api";
-import { logger } from "../util/logging";
+import { credentialLogger, success } from "../util/logging";
```

Comment:
`success` is confusingly named for me, I'm not sure what it does, but it seems like it formats log output - can we come up with a more descriptive name?